### ISSUE TITLE
A quick question about Inet1k acc

### ISSUE THREAD

**Issue by HaiyuWu** [rbalestr-lab/lejepa#35]
Dear Prof. Randall Balestriero,

In the abstract,  "As an example, using imagenet-1k for pretraining and linear evaluation with frozen backbone, LeJEPA reaches 79% with a ViT-H/14.", I wonder how many epochs did you train for getting this accuracy? Is LeJEPA possible to outperform the DINO or I-JEPA on ImageNet?
I am trying to follow this amazing work, but I find it is kinda difficult to reimplement or get the full version of the training processes of the basic experiments.

Thank you.
